xml2rfc/util/name.py

```diff
@@ -34,11 +34,14 @@ def short_author_name_parts(a):
     else:
         fullname = a.get('fullname') or ''
         if fullname:
-            if len(fullname.split())>1:
-                parts = fullname.split()
-                initials = ' '.join([ "%s."%n[0].upper() for n in parts[:-1] ])
-                surname  = parts[-1]
-                parts = [initials, surname ]
+            if is_script(fullname, 'Latin'):
```

Comment:
Maybe this isn't important, as we are pivoting away from the use of initials, but I'm not sure that a Latin writing system is a necessary indicator for the use of basic initials heuristics.  E.g., Cyrillic or Greek use the same initials heuristics (I think), while in general that is of course not true.  So I think I'd like to have some documentation what exactly this change does.

Reply:
I'm not sure where we would put such documentation - ultimately this is something the style guide will describe and may that's the documentation? From an implementation point of view, this is a rats-nest, and this fix is showing to get us what we need for the short term (an RFC containing a Japanese name is on the threshold of release). I suggest we approach this with incremental changes as real cases arrive.

Reply:
Works for me.

There should be something that I can tell kramdown-rfc users about what these attributes mean and how they can be successfully used; this is what I meant with "documentation".  That information would probably live in Github issues for quite a while.